device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/MessagePropertyTest.java

```diff
@@ -131,20 +111,6 @@ public void isValidAppPropertyReturnsTrueForValidAppProperty()
         assertThat(testIsValidAppProperty, is(expectedIsValidAppProperty));
     }
 
-    // Tests_SRS_MESSAGEPROPERTY_11_007: [The function shall return true if and only if the name and value only use characters in: US-ASCII and the name is not a reserved property name.]
-    @Test
-    public void isValidAppPropertyReturnsFalseForInvalidAppProperty()
-    {
-        final String name = "test-name";
-        final String illegalValue = "परीक्षण"; // Unicode is not supported in MessageProperty value
```

Comment:
whoa, who added hindi in here!

Reply:
Haha, I'm not sure! @prmathur-microsoft perhaps?